content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -878,27 +918,32 @@ a known `resourceVersion` is preferable since it can achieve better performance
-| resourceVersionMatch unset            | limit="n", continue="<token>" | Continue Token, Exact | Invalid, treated as Continue Token, Exact | Invalid, HTTP `400 Bad Request`        |
-| resourceVersionMatch=Exact[^1]        | limit unset                   | Invalid               | Invalid                                   | Exact                                  |
-| resourceVersionMatch=Exact[^1]        | limit="n", continue unset     | Invalid               | Invalid                                   | Exact                                  |
-| resourceVersionMatch=NotOlderThan[^1] | limit unset                   | Invalid               | Any                                       | Not older than                         |
-| resourceVersionMatch=NotOlderThan[^1] | limit="n", continue unset     | Invalid               | Any                                       | Not older than                         |
-
-[^1]: If the server does not honor the `resourceVersionMatch` parameter, it is treated as if it is unset.
-| paging                          | resourceVersion unset | resourceVersion="0"                            | resourceVersion="{value other than 0}" |
-|---------------------------------|-----------------------|------------------------------------------------|----------------------------------------|
-| limit unset                     | Most Recent           | Any                                            | Not older than                         |
-| limit="n", continue unset       | Most Recent           | Any                                            | Exact                                  |
-| limit="n", continue="\<token\>" | Continue Token, Exact | Invalid, but treated as Continue Token, Exact  | Invalid, HTTP `400 Bad Request`        |
+| resourceVersionMatch unset            | limit=\<n\>, continue unset     | Most Recent           | Any                                       | Exact                                  |
+| resourceVersionMatch unset            | limit=\<n\>, continue=\<token\> | Continue Token, Exact | Invalid, treated as Continue Token, Exact | Invalid, HTTP `400 Bad Request`        |
+| resourceVersionMatch=Exact<sup>\*</sup>        | limit unset                   | Invalid               | Invalid                                   | Exact                                  |
+| resourceVersionMatch=Exact<sup>\*</sup>        | limit=\<n\>, continue unset     | Invalid               | Invalid                                   | Exact                                  |
+| resourceVersionMatch=NotOlderThan<sup>\*</sup> | limit unset                   | Invalid               | Any                                       | Not older than                         |
+| resourceVersionMatch=NotOlderThan<sup>\*</sup> | limit=\<n\>, continue unset     | Invalid               | Any                                       | Not older than                         |
+{{< /table >}}
+
+<br/>
```

Comment:
Hi @tengqm .
I think the table renders but I suggest using Markdown.
I gave a LGTM because the other formatting changes look good.
The site's styling of tables could use updating (table margins or padding so that the text following a table
is spaced appropriately).
I realize it takes time to create the table formatting.
Is it possible to remove the `sup`, `small`, and `br` HTML elements from the table and use plaintext or Markdown?

Reply:
I could have avoided the HTML hack if only I know some equivalent Markdown syntax for footnotes after a table. Customizing the CSS for tables across the site is beyond this PR.
Let me try abandon the superscript styling then.